test/tests/functional/pbs_maintenance_reservations.py

```diff
@@ -493,28 +510,30 @@ def test_maintenance_reconfirm_reservation_and_run(self):
         reservations. Test if the jobs will run on correct nodes.
         Two moms (-p "servers=M1,moms=M1:M2") are needed for this test.
         """
-        if len(self.moms) != 2:
-            cmt = "need 2 mom hosts: -p servers=<m1>,moms=<m1>:<m2>"
-            self.skip_test(reason=cmt)
```

Comment:
Doesn't this test still need 2 moms?  You are dereferencing self.moms.values()[1] further down.  That'll cause an IndexError unless there are 2 moms.  Or does the @requirements() decorator take care of this and make this skip unnecessary?

Reply:
@requirements decorator won't run the test if we give less than two moms.